sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_path_client.hpp

```diff
@@ -34,7 +34,7 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     static DataLakePathClient CreateFromConnectionString(
         const std::string& connectionString,
         const std::string& fileSystemName,
-        const std::string& path,
+        const std::string& pathName,
```

Comment:
pathname? what 's a path name?

Reply:
Will change it into `dataLakePathName` to be more precise.

Reply:
No, it was `path` and you changed it to `pathName`. What is a path name? I know file name, directory name, they're comonly used words. I've hardly ever seen something called `path name`. Can you share some reference?

Reply:
So what do you propose we call 'The name of a DataLake Path'?

Reply:
what's wrong with `path`?

Reply:
I tend to believe that it is misleading but I don't feel strongly about it. If you think this is a good name I'm fine with it.